functions/createHouse/src/types.ts

```diff
@@ -0,0 +1,19 @@
+export type Responsebody = {
```

Comment:
I think in the future we would like to have these types in a common place that UI and API can access to.